run_linux_docker.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-IMAGE=cgra/cgra-flow:demo
+IMAGE=yuqisun/cgra-flow-openroad:latest
```

Comment:
Have you tried to push to cgra/cgra-flow? Is that possible?

Reply:
Should be impossible, cannot push image to other's repo.